src/lazyload-image.directive.ts

```diff
@@ -1,6 +1,6 @@
-import { Observable } from 'rxjs/Observable';
-import { ReplaySubject } from 'rxjs/ReplaySubject';
-import { switchMap, debounceTime } from 'rxjs/operators';
+import { Observable } from 'rxjs';
+import { ReplaySubject } from 'rxjs';
```

Comment:
Is there any reason that you removed `/ReplaySubject`? This forces the user to use tree shaking, otherwise will the whole rxjs-library be imported.

Reply:
I tried leaving the import statements alone, but I got compile errors.

![image](https://user-images.githubusercontent.com/2002030/38516983-c9fdf1b2-3bf5-11e8-9e62-c3a377a9bbbf.png)

I believe the newest version of rxjs requires the import statements to be refactored. I'm not sure how they're getting around the treeshaking issue, or if they just assume everyone will have treeshaking turned on.

https://docs.google.com/document/d/12nlLt71VLKb-z3YaSGzUfx6mJbc34nsMXtByPUN35cg/edit#heading=h.9gxcyk6xxdc8
https://github.com/ReactiveX/rxjs/blob/master/CHANGELOG.md